docs/install-on-aws.md

```diff
@@ -0,0 +1,233 @@
+<!--[metadata]>
++++
+title = "Build a Swarm cluster for production"
+description = "Deploying Swarm on AWS EC2 AMI's in a VPC"
+keywords = ["docker, swarm, clustering, examples, Amazon, AWS EC2"]
+[menu.main]
+parent="workw_swarm"
++++
+<![end-metadata]-->
+
+
+# Build a Swarm cluster for production
+
+This example shows you how to deploy a high-availability Docker Swarm cluster. Although this example uses the Amazon Web Services (AWS) platform, you can deploy an equivalent Docker Swarm cluster on many other platforms.
+
+The Swarm cluster will contain three types of nodes:
+- Swarm manager
+- Swarm node (aka Swarm agent)
+- Discovery backend node running consul
+
+This example will take you through the following steps: You establish basic network security by creating a security group that restricts inbound traffic by port number, type, and origin. Then, you create four hosts on your network by launching EC2 instances, applying the appropriate security group to each one, and installing Docker Engine on each one. You create a discovery backend by running an consul container on one of the hosts. You create the Swarm cluster by running two Swarm managers in a high-availability configuration. One of the Swarm managers shares a host with consul. Then you run two Swarm nodes. You communicate with the Swarm via the primary manager, running a simple hello world application and then checking which node ran the application. To finish, you test high-availability by making one of Swarm managers fail and checking the status of the managers.
+
+For a gentler introduction to Swarm, try the [Evaluate Swarm in a sandbox](https://docs.docker.com/swarm/install-w-machine/) page.
+
+<This example doesn't use Amazon's EC2 Container Service (ECS)>
+
+## Prerequisites
+
+- An Amazon Web Services (AWS) account
+- Familiarity with AWS features and tools, such as:
+  - EC2 Dashboard
+  - VPC Dashboard
+  - VPC Security groups
+  - Connecting to an EC2 instance using SSH
+
+## Establish basic network security
+
+You create basic network security by restricting the types of inbound traffic that can reach the hosts on your network. To accomplish this, you create a security group and add rules to it. Each rule specifies the type, protocol, port range, and source of the traffic that is allowed to reach your hosts. This security group excludes all other inbound traffic. It also has a rule that allows all outbound traffic.
```

Comment:
As a reader, I'd have to get to the 3rd paragraph before I could understand this only applies to AWS not Swarm in general.  Can you clarify that in the first para please. Also, two paragraphs feels like a heavy approach considering this is about Swarm.  Please compress this to a single intro para. Link out to AWS.

If you want to open a new window, you can use HTML links in Markdown. You asked me about outlinks a couple of days ago ...I'd forgotten this technique.

`<a href="https..." target="_blank>blah</a>`